courses/free-courses-vi.md

```diff
@@ -39,23 +41,26 @@
 
 ### Android
 
-* [Lập trình Android - Android Widgets - Các điều khiển](https://www.youtube.com/watch?v=2xDWeWllzWg&list=PLv6GftO355At6jjYThbMn-5r164GJ5Vyb) - ZendVN
-* [Lập trình Android - Menu - Context Menu - Dialog](https://www.youtube.com/watch?v=vS_InAnGUFA&list=PLv6GftO355Avjf5iuNbEUsIZbltzDEuIU) - ZendVN
-* [Lập trình Android - Xây dựng bố cục giao diện với Android Layout](https://www.youtube.com/watch?v=oxWDjizEN6g&list=PLv6GftO355AtfPQx7M3dkWgi9KPUB9S0V) - ZendVN
-* [Lập trình Android cơ bản](https://www.youtube.com/watch?v=rcH4NbngPEk&list=PL33lvabfss1wDeQMvegg_OZQfaXcbqOQh) - Kteam
-* [Lập trình Android với Kotlin](https://www.youtube.com/watch?v=2CKuiSp_uOQ&list=PLzrVYRai0riRFcvx8VYTF7fx4hXbd_nhU) - KhoaPham
+* [Lập trình Android - Android Widgets - Các điều khiển](https://www.youtube.com/playlist?list=PLv6GftO355At6jjYThbMn-5r164GJ5Vyb) - ZendVN
+* [Lập trình Android - Menu - Context Menu - Dialog](https://www.youtube.com/playlist?list=PLv6GftO355Avjf5iuNbEUsIZbltzDEuIU) - ZendVN
+* [Lập trình Android - Xây dựng bố cục giao diện với Android Layout](https://www.youtube.com/playlist?list=PLv6GftO355AtfPQx7M3dkWgi9KPUB9S0V) - ZendVN
+* [Lập trình Android A-Z](https://www.youtube.com/playlist?list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS) - KhoaPham
+* [Lập trình Android cơ bản](https://www.youtube.com/playlist?list=PL33lvabfss1wDeQMvegg_OZQfaXcbqOQh) - Kteam
+* [Lập trình Android với Kotlin](https://www.youtube.com/playlist?list=PLzrVYRai0riRFcvx8VYTF7fx4hXbd_nhU) - KhoaPham
+* [Lập trình Android với new Firebase](https://www.youtube.com/playlist?list=PLzrVYRai0riTLPLclyGuByHvZ8_tDZZIr) - KhoaPham
```

Comment:
I have changed the appropriate kind of YouTube links that I added, and I also have changed other links.